include/orhi/impl/dx12/ComPtr.h

```diff
@@ -0,0 +1,17 @@
+/**
+* @project: orhi (OpenRHI)
+* @author: Jian Bang Xu
+* @licence: MIT
+*/
+
+#pragma once
+
+#if defined(_WIN32)
+
+#ifndef WIN32_LEAN_AND_MEAN
```

Comment:
`#ifndef` not necessary I believe. Defining an already defined macro shouldn't be a problem?